package.json

```diff
@@ -8,10 +8,13 @@
     "dev": "concurrently --kill-others-on-fail \"yarn server\" \"yarn client\"",
     "start": "node server.js",
     "heroku-postbuild": "cd client && npm install && npm install --only=dev --no-shrinkwrap && npm run build",
-    "server:start": "wait-on http://localhost:3000"
+    "server:start": "wait-on http://localhost:3000",
+    "cy:run": "node_modules\\.bin\\cypress run -b chrome --headless",
+    "cy:run:sanity": "for /l %x in (1, 1, 10) do yarn cy:run"
```

Comment:
okay, so please create one command for unix systems b/c this is windows iteration

anyway <3

Reply:
Done. Let me know if it works since I have only windows (should work).